packages/TorneloScoresheet/src/hooks/appMode/recordingState.ts

```diff
@@ -155,7 +156,7 @@ export const makeUseGraphicalRecordingState =
 
     const goToArbiterGameMode = (): void => {
       setAppModeState({
```

Comment:
This call is missing the type, and we shouldn't be definining each property, i know this wasnt your code but it should be `{...appModeState, mode: AppMode.ArbiterRecording}` and that way we dont need to keep editing this every time we add a new field to recording mode